src/setuptools_scm/version.py

```diff
@@ -6,11 +6,14 @@
 import os
 
 from .config import Configuration
-from .utils import trace, string_types
+from .utils import trace, string_types, iter_entry_points
 
-from pkg_resources import iter_entry_points
+try:
+    # Newer setuptools/pkg_resources releases just use that
+    from packaging.version import parse as pkg_parse_version
```

Comment:
setuptools uses vendoring of a version thats not broken

Reply:
How is it broken? Setuptools currently use 20.4, the newest version is 20.8

Can you please give me an actionable suggestion for improving this PR?